R/printOutput.R

```diff
@@ -27,7 +27,7 @@ printOutput <- function(string, len1stcol = 67, lenCols = NULL, colSep = "   ")
   string <- templ
   for (i in 1:length(string)) {
     out <- paste0(formatstr(unname(string)[[i]],
-           ifelse(i >= length(lenCols), i + 12, lenCols[length(string) + 2 - i])), colSep, out)
+           ifelse(i >= length(lenCols), i + 12, lenCols[length(string) + 2 - i])), ifelse(is.null(lenCols), "", colSep), out)
```

Comment:
this was the reason the layout was broken in the first place. I hoped that all I did was backward-compatible, but because of the additional colSep, it wasn't. Now, it should be backward compatible again.